index.html

```diff
@@ -243,7 +243,70 @@ <h2 class="categorie-title">
                         <li>CodePen</li>
                         <li>GitLab</li>
                     </ul>
-									</div>
+				</div>
             </li>
         </ul>
-    </section>
\ No newline at end of file
+	</section>
+	<footer class="footer-container">
+			<div class="footer-form">
+					<h6 class="footer-title">
+							I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+					</h6>
+					<form action="https://formspree.io/f/mgedzjkr" method="post">
```

Comment:
- [ ] I could not submit the form to **_FormSpree_**. The problem could be from the id pasted. Also, the method should be `POST`, not `post`. Kindly look into it and make the form submit successfully to **_FormSpree_** when filled.